README.md

```diff
@@ -0,0 +1,13 @@
+# CytoCherryPick
+
+## Welcome to CytoCherryPick
+CytoCherryPick is an open source software that provides a set of tools for the selection of representative images from a set of images.
+
+## Functions of CytoCherryPick
```

Comment:
Very optional consideration: the use of the word "features" here gets overloaded due to the focus area ("image features" etc), but generally in software one can label "useful functionality / functions" as "features". For example, verbally, "CytoCherryPick provides the following software features to the audience...". Doing this also decouples you from the need to distinguish between whether you're providing functions vs classes vs other software components (it might get abstract along the way 🙂 ).
```suggestion
## Features of CytoCherryPick
```